app/templates/components/doi-related-item.hbs

```diff
@@ -0,0 +1,111 @@
+  }}
+    {{el.control
+      onChange=(action "selectRelatedItemType")
+      allowClear=true
+      placeholder="Select related item Type"
+    }}
+  {{/form.element}}
+</div>
+
+<div class="power-select-label">
+  <label for="relationType" class="required-label subtitle">Relation Type</label> 
+</div> 
+<div class="power-select-fragment" data-test-related-relation-type>
+  {{#form.element
+    controlType="power-select"
+    value=fragment.relationType
+    helpText="The type of the Relation."
+    options=relationTypeList
+    disabled=disabled
+    allowClear=true
+    destination=fragment.relationType as |el|
+  }}
+    <el.control
+      @onChange={{action "selectRelationType"}}
+      @placeholder="Select Relation Type" as |item|>
+        {{humanize (underscore item)}}
+    </el.control>
+  {{/form.element}}
+</div>
+
+<DoiRelatedItemIdentifier @model={{model}} @form={{form}} @fragment={{fragment.relatedItemIdentifier}} />
+
+<div class="power-select-label">
+  <label for="relatedItemVolume" class="subtitle">Volume</label> 
+</div> 
+<input type="text"
+    class="form-control related-item-volume-field" placeholder="" value={{fragment.volume}} oninput={{action "updateRelatedItemVolume" value="target.value"}} data-test-related-item-volume />
+
+<div
+  class="help-block help-block-fragment related-item-volume-field">
+  Volume of the related item.
+</div>
+
+<div class="power-select-label">
+  <label for="relatedItemIssue" class="subtitle">Issue</label> 
+</div> 
+<input type="text"
+    class="form-control related-item-issue-field" placeholder="" value={{fragment.issue}} oninput={{action "updateRelatedItemIssue" value="target.value"}} data-test-related-item-issue />
+
+<div
+  class="help-block help-block-fragment related-item-issue-field">
+  Issue number or name of the related item.
+</div>
+
+<div class="power-select-label">
+  <label for="relatedItemNumber" class="subtitle">Number</label> 
+</div> 
+<input type="text"
+    class="form-control related-item-number-field" placeholder="" value={{fragment.number}} oninput={{action "updateRelatedItemNumber" value="target.value"}} data-test-related-item-number />
+
+<div
+  class="help-block help-block-fragment related-item-number-field">
+  Number of the related item e.g. report number of article number.
+</div>
+
+<div class="power-select-label">
+  <label for="relatedItemPublicationYear" class="subtitle">Publication Year</label> 
+</div> 
+<input type="text"
+    class="form-control related-item-publication-year-field" placeholder="" value={{fragment.publicationYear}} oninput={{action "updateRelatedItemPublicationYear" value="target.value"}} data-test-related-item-publication-year />
+
+<div
+  class="help-block help-block-fragment related-item-volume-field">
+  The year when the resource was or will be made publicly available.
+</div>
+
+{{#each fragment.creators as |creator index|}}
+    <DoiRelatedItemCreator @model={{model}} @fragment={{creator}} @form={{form}} @index={{index}} />
```

Comment:
I think both `DoiRelatedItemCreator` and `DoiRelatedItemContributor` should be multiple fields. Maybe that was not clear i nthe product spec. But a related item can have multiple Creators and Contributors. https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#202-creator

Reply:
They both are already (in the model + controller at least) - the only thing missing is the add/remove button setup, because I didn't see it in the Figma file, and since Title was limited to just one (even though more are permitted by the schema), I assumed these were the same. I will add the required code to enable adding extra creators/contributors in the form.

Reply:
Fixed in the latest commit